tests/agent/actions/test_draft_datasource_config.py

```diff
@@ -60,11 +60,87 @@ def test_test_draft_datasource_config_success(
         "great_expectations_cloud.agent.actions.draft_datasource_config_action.create_session"
     )
     session = create_session.return_value
-    response = session.get.return_value
-    response.ok = True
-    response.json.return_value = build_payload(config=datasource_config, id=config_id)
+    get_draft_config_response = session.get.return_value
+    get_draft_config_response.ok = True
+    get_draft_config_response.json.return_value = build_get_draft_config_payload(
+        config=datasource_config, id=config_id
+    )
```

Comment:
Can you use `responses` for HTTP mocking.
https://github.com/getsentry/responses?tab=readme-ov-file#shortcuts

Examples from this repo.

https://github.com/great-expectations/cloud/blob/d366b02288c5068e07a557ee48e1a3c90d68bb68/tests/agent/test_agent.py#L438-L467

https://github.com/great-expectations/cloud/blob/d366b02288c5068e07a557ee48e1a3c90d68bb68/tests/agent/actions/test_list_table_names_action.py#L78-L95

Reply:
Oh man, much cleaner. Thanks for the suggestion/reminder! Addressed in https://github.com/great-expectations/cloud/pull/305/commits/e71792cce9f48bd427bfd96c35db7709d772e506